lambdas/src/handlers/session-handler.ts

```diff
@@ -54,6 +54,8 @@ export class SessionLambda implements LambdaInterface {
     public async handler(event: APIGatewayProxyEvent, context: any): Promise<APIGatewayProxyResult> {
         try {
             const deserialisedRequestBody = JSON.parse(event.body as string);
+            logger.info("Session lambda triggered with event", deserialisedRequestBody);
```

Comment:
What does the structure of `deserialisedRequestBody` look like, and is there any PII information in it?

Reply:
Oh that's a really good point

Reply:
I don't think so, but this really points to the reason why we only log specific things, like I forgot completely about PII if we start to do this kind of logging one day something we don't want would make it through

Reply:
If you think there's a risk then I can strip out all the event printing and just stick to the status / progress logging